src/main/java/stsjorbsmod/powers/OldBookPower.java

```diff
@@ -0,0 +1,65 @@
+package stsjorbsmod.powers;
+
+import com.megacrit.cardcrawl.actions.common.RemoveSpecificPowerAction;
+import com.megacrit.cardcrawl.cards.AbstractCard;
+import com.megacrit.cardcrawl.cards.DamageInfo;
+import com.megacrit.cardcrawl.core.AbstractCreature;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.powers.AbstractPower;
+import stsjorbsmod.actions.ExhumeCardsAction;
+import stsjorbsmod.util.CardMetaUtils;
+
+public class OldBookPower extends CustomJorbsModPower {
+    public static final StaticPowerInfo STATIC = StaticPowerInfo.Load(OldBookPower.class);
+    public static final String POWER_ID = STATIC.ID;
+
+    private AbstractCard card;
+
+    public OldBookPower(final AbstractCreature owner, AbstractCard card) {
+        super(STATIC);
+
+        this.isTurnBased = false;
+        this.card = card;
+        this.owner = owner;
+
+        updateDescription();
+    }
+
+    @Override
+    public int onAttacked(DamageInfo info, int damageAmount) {
```

Comment:
I think `onAttacked` isn't quite late enough of a trigger for us to use here; it's going to happen before Torii, doesn't happen quite late enough for every edge case; I think the most serious interactions this would order incorrectly with are Tungsten Rod, Torii, and Spirit Shield. I think probably the way to go here would be to modify SpiritShieldPower to use `OnPlayerHpLossPowerSubscriber` and use `onPlayerHpLoss` then have `Old Book` use `onLoseHp`.

Reply:
Jorbs just suggested that he'd prefer Old Book to trigger *after* Lizard Tail and Fairy in a Bottle; that makes it a little more awkward, since both of those don't trigger until well after damage is applied, but Old Book wants to be cancelling the damage the attack *would* apply. The options would be:

* Patch in some sort of `OnPlayerDeathToPrevent` hook in `AbstractPlayer.damage` just after Lizard Tail and Fairy in a Bottle, and have it heal the player for as much damage as they just took when triggered
* Keep it like this comment originally suggested, but have it check whether Fairy/Lizard *would* be applying and skip itself if it thinks they will apply later

I think I like the second option more, even though it requires a gross amount of hardcoding for Mark of the Bloom and the other 2 effects, because the former option creates a weird/confusing behavior in combination with powers/etc that affect healing (eg, it won't do the right thing if Mark of the Bloom is active)